docs/products/postgresql/howto/migrate-aiven-db-migrate.rst

```diff
@@ -1,32 +1,38 @@
+It supports both logical replication, and using a dump and restore process. 
+Logical replication is the default method which keeps the two databases synchronized until the replication is interrupted. 
+If the preconditions for logical replication are not met for a database, the migration falls back to using ``pg_dump``.
 
-==================      =======================================================================
-Variable                Description
-==================      =======================================================================
-``SRC_HOSTNAME``        Hostname for source PostgreSQL connection
-``SRC_PORT``            Port for source PostgreSQL connection
-``SRC_DATABASE``        Database Name for source PostgreSQL connection
-``SRC_USERNAME``        Username for source PostgreSQL connection
-``SRC_PASSWORD``        Password for source PostgreSQL connection
-``SRC_SSL``             SSL setting for source PostgreSQL connection
-``DEST_PG_NAME``        Name of the destination Aiven PostgreSQL service
-``DEST_PG_PLAN``        Aiven Plan for the destination Aiven PostgreSQL service
-==================      =======================================================================
+.. Note::
+    You can use logical replication when migrating from AWS RDS PostgreSQL 10+, whereas the Google Cloud Platform's PostgreSQL for CloudSQL does not support it.
+
+.. Warning::
+    Running a logical replication migration twice on the same cluster will create duplicate data. Logical replication also has `limitations <https://www.postgresql.org/docs/current/logical-replication-restrictions.html>`_ on what it can copy.
+
+What you'll need
+''''''''''''''''
+    
+* The source server is publicly available or there is a virtual private cloud (VPC) peering connection between the private networks.
+* You have a user account with access to the destination cluster from an external IP, as configured in ``pg_hba.conf`` on the source cluster.
 
-Set ``wal_level`` to ``logical``
-''''''''''''''''''''''''''''''''
+In order to use the **logical replication** method, you'll need the following:
+    
+* PostgreSQL version is 10 or higher.
+* Superuser access credentials to the source cluster or the ``aiven-extras`` extension installed. The extension allows you to perform publish/subscribe-style logical replication without a superuser account, and it is preinstalled on Aiven for PostgreSQL servers. See `Aiven Extras on GitHub <https://github.com/aiven/aiven-extras>`_.
+* An available replication slot on the destination cluster for each database migrated from the source cluster.
+* ``wal_level`` setting on the source cluster to ``logical``.
 
-As per :ref:`aiven-db-migrate-migration-requirements`, to perform a migration using ``aiven-db-migrate``, the ``wal_level`` on the source cluster needs to be set to ``logical``.
 
-To review the current ``wal_level`` run the following command on the source cluster via ``psql``::
+Setting the ``wal_level`` to ``logical``
```

Comment:
Let's use "Set" rather than "Setting" here to fit the style used elsewhere.

Reply:
ok